backend/data_validation/data_val_2022.py

```diff
@@ -126,3 +134,29 @@ def check_for_missing_shooting_zones(
                 f"In {match_key}, {team_number} MISSING TELEOP SHOOTING ZONES",
                 error_type=ErrorType.MISSING_DATA,
             )
+
+    def check_for_auto_great_than_6(
+            self,
+            match_key: str,
+            team_number: int,
+            auto_lower_hub: int,
+            auto_upper_hub: int,
+            auto_misses: int,
+    ) -> None:
+        """
+        Checks if robot shot balls in autonomous and scouter states robot shot more than 6 cargo.
+
+        :param match_key: Key of match that was scouted.
+        :param team_number: Number of team that was scouted (eg 4099).
+        :param auto_lower_hub: Number of balls shot into the lower hub during Autonomous.
+        :param auto_upper_hub: Number of balls shot into the upper hub during Autonomous.
+        :param auto_misses: Number of balls missed when shooting during Autonomous.
+        :return:
+        """
+        # Checks auto ball shot count
+        balls_shot_in_auto = auto_lower_hub + auto_upper_hub + auto_misses
+        if balls_shot_in_auto > 6:
+            self.add_error(
+                f"In {match_key}, {team_number} INCORRECT AUTO SHOT COUNT",
+                error_type=ErrorType.INCORRECT_DATA,
```

Comment:
i don't think this is INCORRECT_DATA it should be a warning (bc anything higher than 6 is unlikely but still possible).